openpype/modules/kitsu/plugins/publish/integrate_kitsu_note.py

```diff
@@ -0,0 +1,42 @@
+# -*- coding: utf-8 -*-
+import gazu
+import pyblish.api
+
+
+class IntegrateKitsuNote(pyblish.api.ContextPlugin):
+    """Integrate Kitsu Note"""
+
+    order = pyblish.api.IntegratorOrder
+    label = "Kitsu Note and Status"
+    # families = ["kitsu"]
+
+    def process(self, context):
+        # Check if work version for user
+        is_work_version = bool(context.data.get("intent", {}).get("value"))
```

Comment:
So when intent is not available then note is not integrated -> and review is not integrated?

Reply:
Hum, actually this is the opposite. If the intent has no value, it's supposed to be a normal publish. If an intent is set (WIP, test), then the Production tracker is not updated but OP is.

Reply:
I don't think that intent should affect if note or review is integrated at all. Intent can be `"For review"`, `"Final"` or anything else, dynamically created by studio.

Reply:
You mean, it shouldn't affect note or review by default, it's up to the studio to decide to override or not this behavior?

Reply:
Yes, intent is just an information from artist that should be reflected somewhere, like a "comment" with predefined messages. Should not be used as definition of publish flow, at least not by default.

NOTE: Intent is OpenPype specific thing which is not in pyblish-[base](https://github.com/pyblish/pyblish-base)/[lite](https://github.com/pyblish/pyblish-lite)/[qml](https://github.com/pyblish/pyblish-qml).

Reply:
> If the intent has no value, it's supposed to be a normal publish. If an intent is set (WIP, test), then the Production tracker is not updated but OP is.

If really needed then this sounds more like a Admin Settings tweak like a "filter" for when to integrate to Kitsu. For example:
- Settings for "Exclude Intents for Integration to Kitsu" could be set to `{"WIP", "test"}`
- The integrator would just check against that `if intent in exclude_intents: return`.

Reply:
> Should not be used as definition of publish flow, at least not by default.

Okay, will do it this way.

> If really needed then this sounds more like a Admin Settings tweak like a "filter" for when to integrate to Kitsu.
I don't need it currently, with @ClementHector we thought we should do it this way. Then we'll wait for feedback.

Reply:
Publish comments can be integrated into kitsu/ftrach... with the review, right?

Reply:
> Settings for "Exclude Intents for Integration to Kitsu" could be set to

I would say that this integrator plugin should not look for intent at all, but should have mechanism to skip if "some value" is set (or not set).

Like, what if I don't care about intent at all and just want integrate everything, all the time. With this logic I also have to care about intent even if it should not matter.